packages/grafana-ui/src/components/Combobox/Combobox.test.tsx

```diff
@@ -294,5 +337,57 @@ describe('Combobox', () => {
 
       expect(emptyMessage).toBeInTheDocument();
     });
+
+    describe('with a value already selected', () => {
+      const selectedValue = { value: '1', label: 'Option 1' };
+
+      it('shows an empty text input when opening the menu', async () => {
```

Comment:
In this case, do we want to set the placeholder if we already have a selected value?

Reply:
I was in mixed minds about this, but we should match the old behaviour 👍